server/manager/src/main/java/org/apache/accumulo/manager/metrics/fate/FateMetrics.java

```diff
@@ -60,16 +61,10 @@ public class FateMetrics implements MetricsProducer {
   private final String fateRootPath;
   private final long refreshDelay;
 
-  private AtomicLong totalCurrentOpsGauge;
-  private AtomicLong totalOpsGauge;
-  private AtomicLong fateErrorsGauge;
-  private AtomicLong newTxGauge;
-  private AtomicLong submittedTxGauge;
-  private AtomicLong inProgressTxGauge;
-  private AtomicLong failedInProgressTxGauge;
-  private AtomicLong failedTxGauge;
-  private AtomicLong successfulTxGauge;
-  private AtomicLong unknownTxGauge;
+  private final AtomicLong totalCurrentOpsCount = new AtomicLong(0);
+  private final AtomicLong totalOpsCount = new AtomicLong(0);
+  private final AtomicLong fateErrorsCount = new AtomicLong(0);
+  private final EnumMap<TStatus,AtomicLong> txStatusCounters = new EnumMap<>(TStatus.class);
```

Comment:
These might be better named as "gauge(s)" instead of "count" since these are distinct

Reply:
Not sure what you mean by "these are distinct". I changed them from `gauge` to `count` in the variable names as that seemed to make more sense to me since they are not gauge objects but rather a count of something.

Reply:
So this changes them from gauges to counts? Or were they incorrectly named before?

Reply:
I meant that gauges and counts are distinct from each other and shouldn't be named interchangeably

Reply:
So the `AtomicLong` vars that held the number of each tx type had `gauge` in the name. For example `AtomicLong submittedTxGauge`, `AtomicLong inProgressTxGauge`, `AtomicLong failedInProgressTxGauge`. These `AtomicLong` vars were **used** by the `Gauge`s. They weren't neccesarrily _incorrectly_ named, I just think that it was a bit confusing to call them gauges instead of counts, so I changed them.

Reply:
they might not be `Gauge` objects but they are still gauges no? My point is that a counter only goes up, a gauge can go up or down. It seems the way these `AtomicLong`s are used is to represent a gauge not a count

Reply:
> My point is that a counter only goes up, a gauge can go up or down.

When describing micrometer instruments, yes, a `Gauge` can go up or down where as `Counter`s should be only increasing. 

The AtomicLong variables (e.g., submittedTxCount) are used to store the current counts and are referenced by Micrometer Gauges. They aren't Gauge objects themselves, so naming them "count" accurately reflects their role as "a variable tracking the count of something" and imo we shouldn't apply the "rules" associated with how the micrometer instruments are named when naming other variables even if they are used in reference to micrometer instruments. The only "gauge" that I am aware of is a micrometer gauge so naming a `Gauge` object variable with "gauge" seems fine (not the case here) but otherwise it seems confusing to me.